tidb-cloud/release-notes-2022.md

```diff
@@ -8,6 +8,17 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## November 1, 2022
+
+* Developer Tier is upgraded to [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier). Serverless Tier, a fully-managed, auto-scaling deployment of TiDB, is now available. It is still in beta stage and free to use.
+
+    * A Serverless Tier cluster still contains fully functional HTAP ability as Dedicated Tier clusters.
+    * Serverless Tier gives users faster creation time and instantaneous cold start time. Compared with Deverloper Tier, the creation time reduces from minutes to seconds.
+    * Users do not need to worry about deployment topology. Serverless Tier will adjust automatically according to your requests.
+    * Serverless Tier [enforces TLS connection to clusters for the sake of security](tidb-cloud/secure-connections-to-serverless-tier-clusters.md).
+
+  Get started [here](/tidb-cloud/tidb-cloud-quickstart.md).
+
```

Comment:
For the users who have already created a Developer Tier, what changes will happen to their cluster, and when the changes will be made? Would you please add a sentence to tell that? @sunxiaoguang

Reply:
Yes, we definitely should have mentioned that.

Reply:
Added, PTAL @qiancai